modules/oe_search_mock/src/EuropaSearchMockTrait.php

```diff
@@ -61,12 +62,19 @@ protected function getRequestMultipartStreamResources(RequestInterface $request,
   public function getFiltersFromRequest(RequestInterface $request): array {
     $request->getBody()->rewind();
     $boundary = $this->getRequestBoundary($request);
-    $filters = $sort_parts = [];
+    $filters = $sort_parts = $facet_fields_parts = [];
+    $path = $request->getUri()->getPath();
+
     if (!empty($boundary)) {
       $request_parts = $this->getRequestMultipartStreamResources($request, $boundary);
       $request->getBody()->rewind();
       $search_parts = explode("\r\n", $request_parts[0]);
-      $sort_parts = isset($request_parts[1]) ? explode("\r\n", $request_parts[1]) : [];
+      if ($path === EuropaSearchMockServerConfigOverrider::ENDPOINT_SEARCH) {
+        $sort_parts = isset($request_parts[1]) ? explode("\r\n", $request_parts[1]) : [];
+      }
+      elseif ($path === EuropaSearchMockServerConfigOverrider::ENDPOINT_FACET) {
```

Comment:
What should be for requests to another endpoint? Should it be just "else"?
```suggestion
      else {
```